modules/data-ingestion-sa/main.tf

```
@@ -97,7 +99,7 @@ resource "google_service_account" "pubsub_writer_service_account" {
 // Cloud Function service account
 module "cloudfunction_controller_service_account" {
   source  = "terraform-google-modules/service-accounts/google"
-  version = "4.2"
+  version = "4.5.0"
```

```suggestion
  version = "4.5"
```

Please keep the same format of the original version definition